tiledb/bioimg/openslide.py

```diff
@@ -152,7 +152,7 @@ def _read_image(
 
 class TileDBOpenSlideLevel:
     def __init__(self, uri: str):
-        self._tdb = tiledb.open(uri)
+        self._tdb = tiledb.open(uri, attr="intensity")
```

Comment:
Needs to be replaced with the global variable

Reply:
Being ideologically against `DRY` and failing to use `grep` (or similar) is recipe for trouble  :wink:

```
$ ag intensity -ws

tiledb/bioimg/openslide.py
155:        self._tdb = tiledb.open(uri, attr="intensity")

tiledb/bioimg/converters/__init__.py
3:ATTR_NAME = "intensity"

tiledb/bioimg/converters/base.py
350:    attr = tiledb.Attr(name="intensity", dtype=attr_dtype, filters=[compressor])

tests/__init__.py
25:                name="intensity",
```

Reply:
Half a dozen more broken `tiledb.open` calls in tests. Good(?) thing we open the array just for inspecting the schema and don't read any data:
```
$ ag tiledb.open tests 

tests/integration/converters/test_openslide.py
30:    with tiledb.open(str(tmp_path / "l_0.tdb")) as A:

tests/integration/converters/test_ome_tiff.py
32:            with tiledb.open(str(tmp_path / f"l_{i}.tdb")) as A:
53:    with tiledb.open(str(tmp_path / "l_0.tdb")) as A:
56:    with tiledb.open(str(tmp_path / "l_1.tdb")) as A:
59:    with tiledb.open(str(tmp_path / "l_2.tdb")) as A:
164:    with tiledb.open(str(tiledb_path / "l_0.tdb")) as A:

tests/integration/converters/test_ome_zarr.py
25:    with tiledb.open(str(tmp_path / "l_0.tdb")) as A:

tests/unit/test_openslide.py
21:                with tiledb.open(level_path, "w") as A:
```